java/ql/lib/ext/java.nio.model.yml

```diff
@@ -6,6 +6,7 @@ extensions:
       - ["java.nio", "ByteBuffer", False, "array", "()", "", "Argument[this]", "ReturnValue", "taint", "manual"]
       - ["java.nio", "ByteBuffer", False, "get", "", "", "Argument[this]", "ReturnValue", "taint", "manual"]
       - ["java.nio", "ByteBuffer", False, "wrap", "(byte[])", "", "Argument[0]", "ReturnValue", "taint", "manual"]
+      - ["java.nio", "Paths", False, "get", "(URI)", "", "Argument[0]", "ReturnValue", "taint", "manual"] # old PathCreation
```

Comment:
Is this model a typo? As far as I can tell, the `Paths` class does not exist in the [`java.nio`](https://docs.oracle.com/javase/8/docs/api/java/nio/package-summary.html) package.

Reply:
Indeed, and this shouldn't be adding steps anyway, AND we shouldn't have a step and a sink at the same node. All in this row is a nonsense, thanks for catching 😄